charts/datadog/templates/system-probe-configmap.yaml

```diff
@@ -1,3 +1,14 @@
+{{- if .Values.datadog.networkMonitoring.enabled }}
+{{- $version := (.Values.agents.image.tag | toString | trimSuffix "-jmx") }}
+{{- $length := len (split "." $version ) -}}
+{{- if (gt $length 1) }}
+{{- if not (semverCompare "^6.23.0-0 || ^7.23.0-0" $version) -}}
+{{- fail "datadog.networkMonitoring.enabled requires agent >= 7.23.0" }}
+{{- end }}
+{{- end }}
+{{- end }}
```

Comment:
could you use [doNotCheckTag](https://github.com/DataDog/helm-charts/blob/f003f0de258e7670699ed4bb73629f585a9244cf/charts/datadog/values.yaml#L597) here, like it is used [here](https://github.com/DataDog/helm-charts/blob/fa68a6f473c5521a8830db60899af6b2e8ecbe87/charts/datadog/templates/_helpers.tpl#L4)

We have introduced this setting to allow users that build custom agent images which doesn't use SemVer tag to bypass the version check.